prow/cmd/deck/jobs.go

```diff
@@ -217,7 +217,7 @@ func (ja *JobAgent) update() error {
 			ProwJob: j.ObjectMeta.Name,
 			BuildID: buildID,
 
-			Started:     j.Status.StartTime.Format(time.Stamp),
+			Started:     string(j.Status.StartTime.Time.Unix()),
```

Comment:
this will break some code in the submit-queue IIRC

Reply:
So can I use another format layout?

Reply:
IIRC some code I wrote parses this to figure out which jobs ran most recently. We're phasing out the submit-queue but right now you could probably just update the submit-queue to handle whatever format you're using here.

Reply:
you're fine :-)
https://github.com/kubernetes/test-infra/blob/f312a14c4b9637ecd4d3804c69f5fb85e913b774/mungegithub/mungers/submit-queue.go#L743